Dockerfile

```diff
@@ -0,0 +1,37 @@
+FROM  node:alpine
+
+# needed to build (python needed for compiling node-sass, for instance...)
+RUN apk add --no-cache \
+    make \
+    gcc \
+    g++ \
+    less \
+    vim \
+    psmisc \
+    net-tools \
+    git \
+    tig \
+    wget \
+    curl \
+    openssh-client \
+    python \
+    bash 
+
+RUN mkdir /opt/pizza-sync
+COPY . /opt/pizza-sync/
+
+# frontend
+RUN cd /opt/pizza-sync/frontend && yarn && yarn run build:prod && rm -rf node_modules
```

Comment:
actually, the best would be to use a multi-staged dockerfile so that you don't mixup the build environment from the execution environment

Reply:
Great tips, thanks. I will try to do this :)